Configuration/TCA/tx_formtools_requests.php

```diff
@@ -2,12 +2,12 @@
 return [
     'ctrl' => [
         'title' => 'Stored Forms',
-        'label' => 'email',
+        'label' => 'last_name',
```

Comment:
Can you please add `label_alt` and `label_alt_force` to add first_name, too?

Reply:
Done